Alameen688/MyDiary
# #159514938 Implement LFA feedback from audit

#### What does this PR do?
Delivers feedback made by LFA on audit day
#### Description of Task to be completed?
The following should feedback must have been worked on
- Prevent access to pages that require login
- An error page that displays error
- profile picture will not show unless logged in
#### How should this be manually tested

After cloning the repo
Without logging in Go to
 -  /client/list-entry.html
 - /client/edit-entry.html
 - /client/view-entry.html
 - /client/profile.html
should not get an error message saying you are not authorized
#### Any background context you want to provide?
none
#### What are the relevant pivotal tracker stories?
#159514938
